doc/Client-Portal Protocol.md

```diff
@@ -542,24 +556,124 @@ This a format for client reconnects.
        sdpMLineIndex: number(mLineIndex),   // optional in RemovedCandidatesMessage
        candidate: string(candidateSdp)
       }
+
+    object(TransportOptions)::
+      {
+        type: "webrtc" | "quic-p2p",
+        id: string(transportId) | null,  // null will result to create a new transport channel.
+      }
+
+    object(P2PQuicParametersMessage)::
+      {
+       type: "quic-p2p-parameters",
+       clientTransportParameters: object(P2PQuicClientParametersMessage) | undefined,
+       serverTransportParameters: object(P2PQuicServerParametersMessage) | undefined
```

Comment:
A "P2PQuicParametersMessage" has either "clientTransportParameters" or "serverTransportParameters"?

Reply:
Yes, do you think we should combine them together? Depends on browser implementations, these structures are subject to change.

Reply:
I think we can have different types like "quic-p2p-client-parameters" and "quic-p2p-server-parameters", so readers know only one role can be used.